external-crates/move/move-compiler/tests/move_check/naming/unused_fun_tparam.move

```diff
@@ -0,0 +1,18 @@
+module 0x42::unused_fun_tparam {
+
+    public fun unused<T>(): u64 {
```

Comment:
Could you add a test for the `T` being unused in the signature, but used in the body of the function. And might be good to test it in several different places: let-binding ascription, let-binding type args, pack type args, call type args, type annotations, etc.